packages/core/src/discovery/browser.js

```diff
@@ -0,0 +1,261 @@
+import os from 'os';
+import path from 'path';
+import { promises as fs, existsSync } from 'fs';
+import { spawn } from 'child_process';
+import EventEmitter from 'events';
+import WebSocket from 'ws';
+import rimraf from 'rimraf';
+import log from '@percy/logger';
+import install from '../utils/install-browser';
+import Page from './page';
+
+export default class Browser extends EventEmitter {
+  #pages = new Map();
+  #callbacks = new Map();
+  #closed = false;
+  #lastid = 0;
+
+  defaultArgs = [
+    '--enable-features=NetworkService,NetworkServiceInProcess',
+    '--disable-background-networking',
+    '--disable-background-timer-throttling',
+    '--disable-backgrounding-occluded-windows',
+    '--disable-breakpad',
+    '--disable-client-side-phishing-detection',
+    '--disable-component-extensions-with-background-pages',
+    '--disable-default-apps',
+    '--disable-dev-shm-usage',
+    '--disable-extensions',
+    '--disable-features=TranslateUI',
+    '--disable-hang-monitor',
+    '--disable-ipc-flooding-protection',
+    '--disable-popup-blocking',
+    '--disable-prompt-on-repost',
+    '--disable-renderer-backgrounding',
+    '--disable-sync',
+    '--disable-web-security',
+    '--force-color-profile=srgb',
+    '--metrics-recording-only',
+    '--no-first-run',
+    '--no-sandbox',
+    '--enable-automation',
+    '--password-store=basic',
+    '--use-mock-keychain',
+    '--remote-debugging-port=0'
+  ];
```

Comment:
This come from somewhere? I feel a little uneasy about managing this (even though deep down I know this list exists in one of the deps lol)

Reply:
This is coming from puppeteer. Although I would love to have a comment for each flag describing what their purpose is. I know of a handful, and we can do this for all of them in the future (or I can spend some time looking them up for this PR)